examples/examples-gpu/nyc-taxi/rf-rapids-dask.ipynb

```diff
@@ -4,9 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Should we explain what a cudf dataframe is as compared to a plain pandas df?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>yes I think you're right. Added this:</p><p> </p><blockquote>A <code>cudf</code> data frame is similar to a <code>pandas</code> data frame, but it lives in GPU memory and most operations on it are done on the GPU.</blockquote><p> </p><p>https://github.com/saturncloud/examples/pull/58/commits/1885f5335ff6dcc0b20caaef8a354230c517c640</p>